### PR TITLE
Issue #15214: migrated section 5.3 camel case to whole config testing

### PR DESCRIPTION
**Pull request by Zopsss**
#15214

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java [hunk 1]
```diff
@@ -9,10 +9,13 @@ class InputCamelCaseDefined {
   boolean supportsIpv6OnIos;
 
   void XmlHttpRequest() {}
+  // violation above 'Method name 'XmlHttpRequest' must match pattern'
 
   void YouTubeImporter() {}
+  // violation above 'Method name 'YouTubeImporter' must match pattern'
 
   void YoutubeImporter() {}
+  // violation above 'Method name 'YoutubeImporter' must match pattern'
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java:18]: kept these extra violations intentionally, it was to show that these method names are not part of camel case section.

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java:18]: what check does this violation ?
not clear why there is violation here

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java:18]: oh I got it .... name should be started from lowercase character

**Review by romani — APPROVED** [review]
ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**